ytools/y2tool/rubocop_yast_style.yml

```diff
@@ -39,3 +43,11 @@ Style/NegatedIf:
 # use "raise" instead of "fail"
 Style/SignalException:
   EnforcedStyle: only_raise
+
+# do not force %r
+Style/RegexpLiteral:
+  Enabled: false
+
+# Forbid and/or usage in conditionals
+Style/AndOr:
```

Comment:
I'd leave the upstream default (banning them everywhere). IHMO it's confusing and used rarely.